app/styles/modules/_password-row.scss

```diff
@@ -201,19 +201,13 @@
 
 .show-password-label:hover,
 .show-password-label:active,
-.show-password-label:focus,
-.show-password:hover + .show-password-label,
-.show-password:active + .show-password-label,
-.show-password:focus + .show-password-label {
+.show-password-label:focus {
```

Comment:
👍

Reply:
Will this work with keyboard navigation?

Reply:
@shane-tomlinson we don't support keyboard on those things AFAIK, users just jump between password fields